userdocs/userdocs.docc/Articles/Topics/installation.md

```diff
@@ -2,6 +2,9 @@
 
 Install the Swift extension via the VS Code Marketplace.
 
-The Swift extension is supported on macOS, Linux, and Windows.
+The Swift extension is supported on macOS, Linux, and Windows. Here's what you will need to do to get set up:
+1. Install Swift from the [Swift.org website](https://www.swift.org/install).
+2. Install [Visual Studio Code](https://code.visualstudio.com/Download).
+3. Install the Swift extension from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=swiftlang.swift-vscode).
 
-To install, firstly ensure you have [Swift installed on your system](https://www.swift.org/install/). Then [install the Swift extension](https://marketplace.visualstudio.com/items?itemName=swiftlang.swift-vscode). Once your machine is ready, you can get started with the **Swift: Create New Project...** command.
+Once your machine is ready, you can get started with the **Swift: Create New Project...** command.
```

Comment:
would it be worth giving two options here - 1) new project, as you've laid out - but the other potentially "open a Swift project you're already working on"?